cluster-up/cluster/kind-sriov/sriov-components/manifests/patch-node-selector.yaml.in

```diff
@@ -1,3 +1,9 @@
+- op: add
+  path: /spec/template/spec/nodeSelector
+  value: {}
 - op: add
   path: /spec/template/spec/nodeSelector/$LABEL_KEY
   value: "$LABEL_VALUE"
+- op: test
+  path: /spec/template/spec/nodeSelector/$LABEL_KEY
+  value: "$LABEL_VALUE"
```

Comment:
What happens without this test? Isn't the patch operation supposed to fail in case the `add` operation fails?

Reply:
Yes it should fail.
Having this test is additional mean to ensure the desired configuration.